test/erc-1155/ERC1155.js

```diff
@@ -38,7 +38,7 @@ describe('ERC1155 tests', function () {
     const factory = await ethers.getContractFactory(
       Constants.Contract.ERC1155Mock
     )
-    erc1155 = await factory.deploy(uri)
+    erc1155 = await factory.deploy(uri, Constants.GAS_LIMIT_1_000_000)
```

Comment:
Q: do we still need this?
HIP 584 should support a more accurate gas limit response. If not we should follow up on this separately

Reply:
Removing this causes a `Error: cannot estimate gas; transaction may fail or may require manual gas limit`

Reply:
Okay, add an issue to capture returning at some point to understand why it's necessary and if a more appropriate default should be set

Reply:
@ebadiere did we capture a ticket on this? Can you note it here and I can resolve the comment

Reply:
https://github.com/hashgraph/hedera-smart-contracts/issues/392. Yes, here it is.  Not sure how it got lost from the original posting.